packages/lesswrong/client/scripts/checkCloudinaryImageSizes.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+
+/**
+ * Script that runs in-browser, finds <img> tags containing Cloudinary URLs,
+ * and, if the image is over-sized, console-logs a modified image URL that
+ * crops and scales it to the size that is actually used.
+ */
+export const checkCloudinaryImageSizes = () => {
```

Comment:
Was this used for debugging during development?  Assume it can be deleted now

Reply:
Ah yes, that was an accidentally-added file from another branch. Deleted.